Workbooks/Intune Devices/Devices/IntuneDevices.workbook

```diff
@@ -0,0 +1,351 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Intune Devices Activity "
+      },
+      "name": "Intune Devices Activity Title"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## Parameters"
+      },
+      "name": "Text - Parameters"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "e5c8c9bc-d4ae-4434-95d7-d6d2128399cf",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "type": 4,
+            "description": "Time range chosen to show the Number of Devices Trends",
+            "isRequired": true,
+            "value": {
+              "durationMs": 2419200000
+            },
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                },
+                {
+                  "durationMs": 1209600000
+                },
+                {
+                  "durationMs": 2419200000
+                },
+                {
+                  "durationMs": 2592000000
+                },
+                {
+                  "durationMs": 5184000000
+                },
+                {
+                  "durationMs": 7776000000
+                }
+              ],
+              "allowCustom": true
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "label": "Time Range"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "Parameters"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## Number Of Devices Trends\r\n---"
```

Comment:
suggestion: instead of using all these extra text items as headers, i'd suggest using the header field in advanced settings for the queries themselves, for a few (mostly minor) reasons:
1. keeps the headers similar to all the other headers in other places in the portal
2. makes the workbook simpler + smaller, as it doesn't have all the extra text content
3. headers set directly in query/metrics steps are also used as the aria-label on those charts/grids for accessibility.  otherwise, they end up with the names of the step (which you've filled in so that's good)

Reply:
Make sense. Text items are removed and Header title is set instead.